doc/api/stream.md

```diff
@@ -1197,7 +1197,9 @@ print(fs.createReadStream('file')).catch(console.log);
 
 If the loop terminates with a `break` or a `throw`, the stream will be
 destroyed. In other terms, iterating over a stream will consume the stream
-fully.
+fully. 
+Note that by default file iteration would be by the chunks of the size equal
```

Comment:
Thanks for the pull request! If this is a new paragraph, please put a blank line between it and the previous line. If it is not, please remove the newline. Thanks!

Reply:
Not sure what you mean by that. There is a 80 chars per line restriction:
```
Running Markdown linter on docs...
doc/api/stream.md
  1201:153  warning  Line must be at most 80 characters  maximum-line-length  remark-lint
```